packages/web/docs/src/components/landing-page.tsx

```diff
@@ -90,6 +96,8 @@ export function IndexPage(): ReactElement {
         <TeamSection className="mx-4 md:mx-6" />
         <CommunitySection className="mx-4 mt-6 md:mx-6" />
         <ToolsAndLibrariesCards className="mx-4 mt-6 md:mx-6" />
+        <FrequentlyAskedQuestions className="mx-4 md:mx-6" />
+        <GetYourAPIGameRightSection className="mx-4 md:mx-6" />
```

Comment:
later thing but wondering if the parent container can just add the appropriate margin horizontally and flex gap for vertical

Reply:
> container can just add the appropriate margin horizontally

Yup, +1. I didn't know before that the margins are consistent before I implemented those sections (because they aren't in feature tabs)

> flex gap for vertical

Those aren't consistent. It's either `mt-6` or 0, and adding a gap and negative margin feels worse.